src/styles/blocks/buttons.scss

```diff
@@ -0,0 +1,29 @@
+.buttons {
+  @include pageGrid;
+
+  &__config {
+    // grid-column: 1 / -1;
+    cursor: pointer;
```

Comment:
You have two 'cursor: pointer;' declarations for the same element, consider removing one of them.